src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+}
+
+export const Tabs: FC<Props> = ({
+  tabs,
+}) => {
+  const { tabId = 0 } = useParams();
+
+  const selectedTab = tabs.find(tab => tab.id === tabId) || '';
```

Comment:
Now everything is written correctly, but if the `.find` method does not find the required element, it will return `undefined`, which is suitable for further checking the condition of existence, so there is no need to return an empty string.
You can learn more about the `find` method here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find?retiredLocale=uk